x-pack/plugins/security_solution/server/endpoint/routes/metadata/query_builders.ts

```diff
@@ -22,7 +23,11 @@ export async function kibanaRequestToMetadataListESQuery(
   const pagingProperties = await getPagingProperties(request, endpointAppContext);
   return {
     body: {
-      query: buildQueryBody(request, queryBuilderOptions?.unenrolledAgentIds!),
+      query: buildQueryBody(
+        request,
+        queryBuilderOptions?.unenrolledAgentIds!,
+        queryBuilderOptions?.statusAgentIDs!
```

Comment:
Do you think we should pass the queryBuilderOptions into the function, and handle the options inside the function

Reply:
If we get a third, we can explore that